### PR TITLE
Add support to pass parameters to menu translate function

### PR DESCRIPTION
**Pull request by alr2413**
Now you can pass extra parameters to menu translate function

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /composer.lock
 /vendor
 /.idea
+.DS_Store
```
Comment (Shidersz) [inline .gitignore:4]: This is not needed, remove please.

**Review by Shidersz — CHANGES_REQUESTED** [review]

**Comment by lacodimizer** [comment]
Thank you for your merge request. Can you add tests for this case please?

File: .gitignore (hunk 1 above)
Comment (alr2413) [reply .gitignore:4]: Ok, I'll remove it.

**Comment by alr2413** [comment]
> Thank you for your merge request. Can you add tests for this case please?

Yea, I did that.

**Comment by Shidersz** [comment]
@lacodimizer I'm planning to improve the changes introduced by this PR later, the changes on the `resources/lang/en/menu.php` file are not needed, there is a way to add translation at run time with the `addLines()` method of a `Translator` instance. I will get on this when have some free time.

**Pull request merged**